src/Data/Bool/Properties.agda

```diff
@@ -21,14 +25,21 @@ open import Induction.WellFounded using (WellFounded; Acc; acc)
 open import Level using (Level; 0ℓ)
 open import Relation.Binary.Core using (_⇒_)
 open import Relation.Binary.Structures
-  using (IsPreorder; IsPartialOrder; IsTotalOrder; IsDecTotalOrder; IsStrictPartialOrder; IsStrictTotalOrder)
+  using (IsPreorder; IsPartialOrder; IsTotalOrder; IsDecTotalOrder
+  ; IsStrictPartialOrder; IsStrictTotalOrder)
 open import Relation.Binary.Bundles
-  using (Setoid; DecSetoid; Poset; Preorder; TotalOrder; DecTotalOrder; StrictPartialOrder; StrictTotalOrder)
+  using (Setoid; DecSetoid; Poset; Preorder; TotalOrder; DecTotalOrder;
+  StrictPartialOrder; StrictTotalOrder)
```

Comment:
```suggestion
  using (Setoid; DecSetoid; Poset; Preorder; TotalOrder; DecTotalOrder
  ; StrictPartialOrder; StrictTotalOrder)
```
style of splitting across multiple lines is different here to above. Moreover, custom (if not `style-guide`?) dictates aligning the leading `;` on a wrapped line with the leading `(` parenthesis?